packages/blockchain-wallet-v4-frontend/src/scenes/DebitCard/CardDashboard/index.tsx

```diff
@@ -11,6 +11,7 @@ const Iframe = styled.iframe`
 `
 
 const CardDashboard = ({
+  alertActions,
```

Comment:
Instead of passing actions as a param to child components just do it here in this component. The index.tsx files are more for the getting/setting data and the template files are for displaying the data. 
You could do:
```
const alertErrorCallback = useCallback(() => {
  alertActions.displayError(C.DEBIT_CARD_DASHBOARD_LOCK_ERROR)
}, [])

...
<ManageCardBox
  alertErrorHandler={alertErrorCallback}
  ...
/>
```